src/xml_object.ts

```diff
@@ -249,7 +249,20 @@ export class XmlObject implements IXmlSerializable {
                                 } else {
                                     node = doc.createElementNS(schema.namespaceURI, `${schema.prefix ? schema.prefix + ":" : ""}${schema.localName}`);
                                 }
-                                node!.textContent = value;
+                                if (Array.isArray(value)) {
+                                    for (let child of value) {
```

Comment:
[tslint] Identifier 'child' is never reassigned; use 'const' instead of 'let'.

Reply:
Lint are fixed now